app/src/main/java/edu/uw/tcss450/chatapp_group1/ui/home/HomeFragment.java

```diff
@@ -24,6 +25,7 @@
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
+
```

Comment:
remember get rid of extra lines that aren't necessary

Reply:
sure, I will